src/utils/components/EmptyBox/Box.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC, ReactNode } from 'react';
+import classNames from 'classnames';
+
+type BoxProps = {
+  children: ReactNode;
+  className?: string;
+};
+
+const Box: FC<BoxProps> = ({ children, className }) => (
```

Comment:
I am not sure about this. Are we ever gonna need this component? Except using it in `EmptyBox` component.. It looks so simple and unnecessary, I'd create only one component instead.